pkg/app/app.go

```diff
@@ -68,8 +69,24 @@ func NewApp(
 	es encryption.Service,
 	dbs database.Service,
 	webStaticFilesPath string,
+	corsAllowedOrigins []string,
 	webRTCConfig config.WebRTCConfig) *App {
-	return &App{im, am, oc, es, dbs, webStaticFilesPath, buildInfraCfg(webRTCConfig.STUNServers)}
+	return &App{im, am, oc, es, dbs, webStaticFilesPath, corsAllowedOrigins, buildInfraCfg(webRTCConfig.STUNServers)}
+}
+
+func (c *App) AddCorsHeaderIfNeeded(w http.ResponseWriter, r *http.Request) {
+	origin := r.Header.Get("Origin")
+
```

Comment:
nit: remove the emtpy lines: 79, 83, 124, 127.

Reply:
Removed Removed unnecessary lines.